src/main/java/factionsystem/Subsystems/CommandSubsystem.java

```diff
@@ -613,6 +613,13 @@ public boolean interpretCommand(CommandSender sender, String label, String[] arg
                     return true;
                 }
 
+                // gate management commands
+                if (args[0].equalsIgnoreCase("gate") || args[0].equalsIgnoreCase("gt")) {
+                	GateCommand command = new GateCommand(main);
```

Comment:
The details of the gate command needs to be added to the output of the the /mf help command.